archivebox/parsers/pocket_api.py

```diff
@@ -96,7 +99,7 @@ def should_parse_as_pocket_api(text: str) -> bool:
 
 
 @enforce_types
-def parse_pocket_api_export(input_buffer: IO[str], **_kwargs) -> Iterable[Link]:
+def parse_pocket_api_export(input_buffer: IO[str], **_kwargs) -> Iterable[Model]:
```

Comment:
not `Iterable[Snapshot]`?

Reply:
Cannot do that. Django will not let me import `Snapshot` directly, even with the typing condition. This was the closest I could think of.

Reply:
ok